src/renderer/renderElement.ts

```diff
@@ -964,6 +969,8 @@ export const renderElementToSvg = (
           image.setAttribute("width", "100%");
           image.setAttribute("height", "100%");
           image.setAttribute("href", fileData.dataURL);
+          image.style.transform =
+            element.scale[0] === -1 ? `translateX(${width}px) scaleX(-1)` : "";
```

Comment:
applying on the image would flip all instances of the image since the idea behind this is having just a single image element per image instance on canvas for file size reasons

Reply:
good catch 👍